### PR TITLE
Add: failed pod support

### PR DESCRIPTION
**Pull request by omrisavra**

### CODE REVIEW

File: docs/Learn/Annotations.md [hunk 1]
```diff
@@ -108,9 +108,10 @@ The following values can be used to enrich the URL:
 |${service} | Service Name|
 |${namespace} | Namespace Name|
 |${cluster} | Cluster Name|
+|${failedPod} | Failed pod name connected to event* |
```
Comment (guymenahem) [inline docs/Learn/Annotations.md:111]:
```suggestion
|${failedPod} | The pod name of a failed pod that triggered this health event.* |
```

File: docs/Learn/Annotations.md [hunk 2]
```diff
@@ -141,7 +142,8 @@ spec:
 |{$yaml[spec.my_replicas]}                       | undefined     | path doesn't exist                      |
 |{$yaml[spec.tamplate.spec.containers]}          | undefined     | path doesn't resolve to an actual value |
 
-*Dates can be crasfted using the display guidelines of date-fns https://date-fns.org/v2.25.0/docs/format
+*Custom link with failedPod won't appear for events which aren't connected to a failedPod
```
Comment (guymenahem) [inline docs/Learn/Annotations.md:145]:
```suggestion
*Custom links with a failed pod name will be created on health events only. 
```

File: docs/Learn/Annotations.md [hunk 3]
```diff
@@ -141,7 +142,8 @@ spec:
...
+**Dates can be crasfted using the display guidelines of date-fns https://date-fns.org/v2.25.0/docs/format
```
Comment (guymenahem) [inline docs/Learn/Annotations.md:146]: I think you got a typo in crafted

**Review by guymenahem — CHANGES_REQUESTED** [review]

File: docs/Learn/Annotations.md (hunk 3 above)
Comment (omrisavra) [reply docs/Learn/Annotations.md:146]: Cool, not my typo will fix it :)

**Review by guymenahem — APPROVED** [review]

**Pull request merged**